src/services/db-manager.ts

```diff
@@ -39,7 +39,8 @@ function randomHexValue() {
 }
 
 export function genDbId(dbAlias: string) {
-  return config.auth ? `_${randomHexValue()}` : dbAlias;
+  // TODO: Find a better way to distinguish multi-user vs single-user situations
```

Comment:
maybe just check the configuration name directly?

Reply:
Not a good idea because if we want to add another operating environment in the future that should be multi-user the code wouldn't catch that.

But I don't know why I didn't realize that I should just *create* a multi-user flag rather than relying on the fact that currently any usage of Sentry is a multi-user environment, so I'm fixing that.